subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/WatchingNotSupportedVirtualFileSystem.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.vfs.impl;
+
+import org.gradle.internal.vfs.WatchingVirtualFileSystem;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+public class WatchingNotSupportedVirtualFileSystem extends AbstractDelegatingVirtualFileSystem implements WatchingVirtualFileSystem {
```

Comment:
I think this one should be called `NonWatchingFileSystem`.

Should the implemented interface be called something like `WatchAwareFileSystem` instead, too?

Reply:
Renamed.